clearnode/channel.go

```diff
@@ -114,3 +114,22 @@ func CheckExistingChannels(tx *gorm.DB, wallet, token string, chainID uint32) (*
 
 	return &channel, nil
 }
+
+type ChannelAmountSum struct {
+	Count int             `gorm:"column:count"`
+	Sum   decimal.Decimal `gorm:"column:sum"`
+}
+
+func GetChannelAmountSumByWallet(tx *gorm.DB, senderWallet string) (ChannelAmountSum, error) {
```

Comment:
Could you explain why have you created a separate structure, and not just returning 2 separate values?

Reply:
I am just not sure it provides more benefits then disadvantages.
On one side the structure is an object that hosts all the relevant fields and it makes sense to create it when we need to include a lot of fields or it is passed a lot.
However here it only includes 2 fields and is not passed around a lot.
That is why I am not confident it is the best to include it here.
Maybe you could explain more why is it cleaner on both sides?